clients/privacy-center/packages/fides-consent/rollup.config.js

```diff
@@ -23,8 +23,13 @@ const generateConsentConfig = () => {
    * @type {import('../../types/config').Config}
    */
   const privacyCenterConfig = require("../../config/config.json");
-  const privacyCenterOptions =
-    privacyCenterConfig.consent?.consentOptions ?? [];
+  const isV1ConsentConifg = typeof privacyCenterConfig.consent?.button === "undefined";
+  let privacyCenterOptions = [];
+  if (isV1ConsentConifg) {
```

Comment:
small typo :)

```suggestion
  const isV1ConsentConfig = typeof privacyCenterConfig.consent?.button === "undefined";
  let privacyCenterOptions = [];
  if (isV1ConsentConfig) {
```

Reply:
this could probably be a ternary as well, i.e.

```js
const privacyCenterOptions = isV1ConsentConfig ? privacyCenterConfig.consent?.consentOptions : privacyCenterConfig.consent?.page.consentOptions;

if ( !privacyCenterOptions || privacyCenterOptions.length === 0) {
  ...
}
```

Reply:
Thanks!